certbot-dns-rfc2136/certbot_dns_rfc2136/dns_rfc2136_test.py

```diff
@@ -69,6 +69,30 @@ def test_valid_algorithm_passes(self):
         self.auth.perform([self.achall])
 
 
+class DNSLookupTest(unittest.TestCase):
+    def setUp(self):
+        from certbot_dns_rfc2136.dns_rfc2136 import _RFC2136Client
+        self.client = lambda x: _RFC2136Client(x, NAME, SECRET, dns.tsig.HMAC_MD5)
+
+    def test_ipv4(self):
+        self.client("127.0.0.1")
+
+    # Travis Ci does not support IPv6 yet, so this test is disabled
```

Comment:
It has been a while since I've written Python unit tests, so this could be a crazy suggestion, but...

Would it be possible to use something like `@unittest.skip("Travis CI does not support IPv6 yet, so this test is disabled")` instead of just commenting it out? Hopefully this would cause the fact that the unit test is skipped to be reported in the output, to help remind anyone making changes in this area that the test is being skipped.

Even better might be to use something like `@unittest.skipIf(???, "This system does not support IPv6, so this test will be skipped")`, assuming there's a simple conditional check for IPv6 support you can substitute for the `???`. If that works like I'm imagining, it would let people easily test IPv6 locally (on systems that support it) and automatically start testing it in Travis CI if support is ever added there.

Reply:
So determining whether IPv6 is supported or not in python is possible, but rather complex (and there are various ways how IPv6 might not be available, such as it is supported by the kernel but maybe just not configured on the loopback interface) and I would like to avoid this for now.

However there is an alternative solution:
```
    @unittest.skipIf("TRAVIS" in os.environ,
                     "IPv6 is not always supported in Travis, IPv6 test skipped")
```

This will disable the IPv6 check on Travis for now. As soon as the overall Travis configuration/environment will support IPv6, we can remove this and enable the test again.

Also, I improved the general tests a bit by comparing the server property with the expected values.